pkgs/tools/package-management/nixops/nixops-dns.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, stdenv
+, buildGoPackage
+, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "nixops-dns";
+  version = "HEAD";
```

Comment:
I tagged it with `v1.0` https://github.com/kamilchm/nixops-dns/releases/tag/v1.0